exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/text_ai/txaie_preprocessing.ipynb

```diff
@@ -0,0 +1,3564 @@
+{
```

Comment:
<p>Calling this notebook from here doesn't look very pretty. You will get a piece of UI in the output asking you to enter the secret store credentials. The user will be unsure what to do with it. It confused me as well.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/ai-lab/pull/364/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>i changed it, but now it is super easy to miss and just skip this step. not sure i am happy with this solution. alternatively we could call it from here and remove the secret store acces window from the notebook?</p>